pkg/trace/api/pipeline_stats.go

```diff
@@ -25,12 +25,12 @@ const (
 )
 
 // pipelineStatsEndpoint returns the pipeline intake url and the corresponding API key.
-func pipelineStatsEndpoint(cfg *config.AgentConfig) (url *url.URL, apiKey string, err error) {
+func pipelineStatsEndpoint(cfg *config.AgentConfig) (*url.URL, string, error) {
```

Comment:
Why drop the return value names here?

Reply:
That's actually the fix, we don't want to declare the url variable, otherwise `url.Parse` in L33 would call the url's Parse method instead of the `Parse` function from `net/url`